src/components/Navigation/NavigationItems/NavigationItems.js

```diff
@@ -3,12 +3,13 @@ import classes from './NavigationItems.module.css';
 import NavigationItem from './NavigationItem/NavigationItem';
 import * as routeConst from '../../../Shared/Constants/constantRouter';
 const navigationItems = (props) => {
-
     return (
             <ul className= {classes.NavigationItems}> 
-                  <NavigationItem Link={routeConst.ROOT_PATH} exact >Add To Do</NavigationItem>
-                  <NavigationItem Link={routeConst.CHARTS_PATH} >Charts</NavigationItem>
-                  <NavigationItem Link={routeConst.AUTHENTICAT_PATH} >Autenticate</NavigationItem>
+                {props.isAuth ?<NavigationItem Link={routeConst.ROOT_PATH} exact >Add To Do</NavigationItem> :null}
+                 {props.isAuth ? <NavigationItem Link={routeConst.CHARTS_PATH} >Charts</NavigationItem> : null}
```

Comment:
You can use props.isAuth&& <NavigationItem Link={routeConst.ROOT_PATH} exact >Add To Do</NavigationItem>